torch/utils/_python_dispatch.py

```diff
@@ -65,7 +65,12 @@ def enable_torch_dispatch_mode(mode, *, replace=None, ignore_preexisting=False)
 def _wrap_torch_dispatch(f):
     @functools.wraps(f)
     def wrapped(self, *args, **kwargs):
-        with enable_torch_dispatch_mode(self.inner):
+        if hasattr(self, "inner"):
+            inner = self.inner
+        else:
+            inner = None
```

Comment:
nit: getattr(self, "inner", default=None). This logic looks a lot like the one in torch_function :P